bitcoin/chainparams.c

```diff
@@ -79,7 +79,7 @@ const struct chainparams networks[] = {
 			   .bip32_privkey_version = BIP32_VER_TEST_PRIVATE},
      .is_elements = false},
     {.network_name = "signet",
-     .bip173_name = "tb",
+     .bip173_name = "tbs",
```

Comment:
From the issue

>LND for example doesn't recognize these invoices as signet and will not pay them.

I read some time ago a conversation in IRC where if I remember correctly the cause is a name collation between an lnd team decision and others impl.

Maybe this is acceptable behavior and the error here is the nomenclature in lnd?

I don't know if it is right at all, or I remember somethings not true